sdk/sdk-api/src/main/java/mil/emp3/api/utils/EmpBoundingBox.java

```diff
@@ -99,22 +107,32 @@ public double east() {
      * This method returns the western longitude.
      * @return The western logintude in degrees.
      */
+    @Override
     public double west() {
         return this.getWest();
     }
 
     @Override
     public void setNorth(double value) {
+        if (Double.isNaN(value)) {
+            throw new InvalidParameterException("value is not a number.");
```

Comment:
I think you should be using IllegalArgumentException here instead. The javadocs indicate this is more for use in the context of JCA/JCE.

> throw new InvalidParameterException("value is not a number.");